deploy/templates/notificationtemplates/appstudio/idlertriggered/notification.html

```diff
@@ -46,7 +46,7 @@
 
     <p>
         In accordance with the usage terms, we have reduced the number of instances of your
-        application to zero (0). You can restart your application(s) by increasing the number of instances from the User Interface.
+        application {{.Apptype}} {{.Appname}} to zero (0). You can restart your application(s) by increasing the number of instances from the User Interface.
```

Comment:
Let's use CammelCase as we mentioned in the other PR.

Reply:
Done